Unix/etc/omiserver.conf

```diff
@@ -79,3 +79,11 @@ service=<USER>
 # means that each provider runs under it's own omiagent process.
 #agentDebugging=false
 
+# List of authorized and unauthorized user groups, separated by commas.
+# Both local and domain groups can be listed.  For example:
+#    AuthorizedGroups=SCX\scx local admins, SCX\domain users, adm
+#    UnauthorizedGroups=games
+# If both parameters are blank (default), authorization checks are not
+# performed. For more info, see doc/allow-deny-handling.md
```

Comment:
Can you give a fully http path here, in case the customer has never dealt with the source code?